File: TOKI/Book/01-PerkenalanPemrogramanKompetitif/A.LampuDanBotol/1.SolusiSederhana/Main.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long n = sc.nextInt();
        sc.close();

        int divisorCount = 0;
        for(long i = 1; i <= n; i++) {
            if(n%i == 0) {
                divisorCount++;
            }
        }

        if(divisorCount % 2 == 0) {
            System.out.println("lampu mati");
        }else {
            System.out.println("lampu menyala");
        }
    }
}

File: TOKI/Book/01-PerkenalanPemrogramanKompetitif/A.LampuDanBotol/1.SolusiSederhana/main.cpp
#include <iostream>
using namespace std;

int main() {
    long long N;
    cin >> N;

    int divisorCount = 0;
    for(long long i = 1; i <= N; i++) {
        if(N % i == 0) {
            divisorCount++;
        }
    }

    if(divisorCount % 2 == 0) {
        cout << "lamput mati" << endl;
    }else {
        cout << "lampu menyala" << endl;
    }
}